.travis.yml

```diff
@@ -24,7 +24,6 @@ env:
     - RUST_BACKTRACE=1
     - RUST_TEST_THREADS=1
   matrix:
-    -
```

Comment:
I thought this one is neccessary to have env without `RELEASE` set

Reply:
There's nothing wrong with empty yaml list elements, AFAIK. :)

Reply:
> There's nothing wrong with empty yaml list elements, AFAIK.

Ok, if there is a reason, lets keep it. My IDE reported a problem here. I will fix this in the next commit.